libs/application/template-api-modules/src/lib/modules/templates/institution-collaboration/institution-collaboration.module.ts

```diff
@@ -0,0 +1,33 @@
+import { DynamicModule } from '@nestjs/common'
+import { SharedTemplateAPIModule } from '../../shared'
+import { BaseTemplateAPIModuleConfig } from '../../../types'
+import { InstitutionCollaborationService } from './institution-collaboration.service'
+import { INSTITUTION_COLLABORATION_CONFIG } from './config/institutionApplicationServiceConfig'
+import { FileStorageModule } from '@island.is/file-storage'
+
+const recipientEmailAddress =
+  process.env.INSTITUTION_COLLABORATION_RECIPIENT_EMAIL_ADDRESS ??
+  'development@island.is'
+const senderEmailAddress =
+  process.env.INSTITUTION_COLLABORATION_SENDER_EMAIL_ADDRESS ??
```

Comment:
I wonder if it should be the same as the one defined in AWS

```
/k8s/application-system/api/EMAIL_FROM
/k8s/application-system/api/EMAIL_FROM_NAME
/k8s/application-system/api/EMAIL_REPLY_TO
/k8s/application-system/api/EMAIL_REPLY_TO_NAME
```

Reply:
@jeremybarbet Good point, I will look into that.

Reply:
We should be allowed to use these values:
`/k8s/application-system/api/EMAIL_FROM
/k8s/application-system/api/EMAIL_FROM_NAME`
for sender, but I am not sure if we can use `/k8s/application-system/api/EMAIL_REPLY_TO` for the recipient email, I need to do some checking on that.